components/Internship/CommitteeInternshipInfo.js

```diff
@@ -0,0 +1,42 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+import styles from '../../styles/components/Internship/CommitteeInternshipInfo.module.scss';
+
+function CommitteeInternshipInfo(props) {
+    const {committees} = props;
+    const [committeeName, setCommitteeName] = useState('Board');
+    return(
+        <>
+            <div className={styles['logos-line']}>
+                {committees.map(committee =>
+                    <div className={styles['logo-diamond']} key={committee.name} onClick={()=>setCommitteeName(committee.name)}>
```

Comment:
If this is a clickable item, can we make it a `<button>`?

There's more accessibility work that needs to be done here. The clickable elements aren't tabbable, and there's some HTML annotations we need to make; see [this W3 example of a tab implementation](https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-2/tabs.html).